config/config.go

```diff
@@ -23,5 +23,7 @@ func init() {
 
 // Prefixes used in Mist stream names to let us determine whether a given "stream" in Mist is being used
 // for the segmenting or transcoding phase
+const SEGMENTING_PREFIX = "catalyst_vod_"
```

Comment:
Should we just call this "vod"? Technically everything is catalyst now.

Reply:
Good point. Maybe we should save `vod+` prefix for playback.

Created issue https://github.com/livepeer/catalyst-api/issues/51